keyboards/lets_split/keymaps/default/keymap.c

```diff
@@ -24,9 +24,9 @@ enum custom_keycodes {
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 /* Qwerty
  * ,-----------------------------------------------------------------------------------.
- * | Tab  |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |   P  | Bksp |
+ * | Esc  |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |   P  | Bksp |
```

Comment:
I'm not quite sure why this was changed either. Looking at the history at some point I merged an incorrect ASCII art correction into QMK upstream, but it's since been fixed too.

Removing this change.